llarp/service/endpoint.cpp

```diff
@@ -1551,7 +1532,7 @@ namespace llarp
       else
       {
         llarp::LogInfo(Name(), " updating introset");
-        UpdateIntroSet(false);
+        UpdateIntroSet(true);
```

Comment:
no documentation about this change? Is this intentional?

Reply:
i think my local changes got sucked into this pr